examples/amp-story/interactive_img.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html ⚡ lang="es">
+  <head>
+    <meta charset="utf-8">
+    <title>Interactive Image Components Story</title>
+    <link rel="canonical" href="interactive_img.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        font-family: 'Rubik', sans-serif;
```

Comment:
nit: Why are we using Rubik? I don't think I've seen this font in our codebase before.
We typically use Roboto for small text and Poppins for larger text.

Reply:
I'll go ahead and remove this; it's just there because I copy/pasted from the interactives.html example story

Reply:
Oh! I see it now. Thanks for pointing this out. Up to you if you want to remove it or not. 
It might be okay to leave it because you could make your demo story match interactives.html

Reply:
Just for reference, when I had to implement the font loading for Poppins on the interactive components, the best way to test that the font gets loaded properly is if the outside elements are not Poppins (because then the font would not have to be loaded by the components), so I chose another font so the components fail if Poppins is not imported correctly. Feel free to style this story as you wish.